src/components/NewLexemeForm.vue

```diff
@@ -28,6 +28,16 @@ const lexicalCategory = computed( {
 		store.commit( SET_LEXICAL_CATEGORY, newLexicalCategory );
 	},
 } );
+
+const submitMsg = $messages.get( 'wikibaselexeme-newlexeme-submit' );
+const termsOfUseURL = 'https://foundation.wikimedia.org/wiki/Terms_of_Use';
```

Comment:
Hard-coding this doesn’t seem right to me – in Data Bridge, we get it from the custom `wbdatabridgeconfig` API query+meta module, where it comes from parsing the [copyrightpage message](https://www.wikidata.org/wiki/MediaWiki:Copyrightpage) as a title.

Reply:
see latest commit